src/components/Account.vue

```diff
@@ -49,65 +51,46 @@
         display: flex;
         flex-direction: row;
         align-items: center;
-        padding: calc(1.5 * var(--nimiq-size, 8px)) calc(4 * var(--nimiq-size, 8px));
+        justify-content: space-between;
+        padding: calc(1.75 * var(--nimiq-size, 8px)) calc(2 * var(--nimiq-size, 8px));
         box-sizing: border-box;
         flex-shrink: 0;
+        font-size: calc(2 * var(--nimiq-size, 8px));
+        line-height: 1.25;
     }
 
-    .identicon {
-        width: calc(8 * var(--nimiq-size, 8px));
-        height: calc(8 * var(--nimiq-size, 8px));
-        flex-shrink: 0;
-        margin-right: calc(2 * var(--nimiq-size, 8px));
-    }
-
-    .account-description {
-        flex-grow: 1;
-        --lost-width: calc(10 * var(--nimiq-size, 8px));
-        width: calc(100% - var(--lost-width));
-    }
-
-    .label-and-balance {
+    .identicon-and-label {
         display: flex;
         flex-direction: row;
-        justify-content: space-between;
-        font-weight: 600;
-        font-size: calc(2.25 * var(--nimiq-size, 8px));
-        line-height: calc(2.5 * var(--nimiq-size, 8px));
-        margin-bottom: calc(0.375 * var(--nimiq-size, 8px));
+        align-items: center;
+        overflow: hidden;
+        min-width: calc(5.625 * var(--nimiq-size, 8px));
     }
 
-    .editable .label-and-balance {
-        margin-bottom: 0;
+    .identicon {
+        width: calc(5.625 * var(--nimiq-size, 8px));
+        height: calc(5.625 * var(--nimiq-size, 8px));
+        flex-shrink: 0;
+        margin-right: calc(1.5 * var(--nimiq-size, 8px));
     }
 
     .label {
         white-space: nowrap;
+        font-weight: 600;
         overflow: hidden;
-        text-overflow: ellipsis;
     }
 
-    .label >>> input {
-        padding-bottom: calc(0.375 * var(--nimiq-size, 8px));
-    }
-
-    .label >>> .nq-icon {
-        margin-bottom: calc(0.375 * var(--nimiq-size, 8px));
+    .label div {
+        opacity: 0.7;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        padding-left: calc(1 * var(--nimiq-size, 8px));
```

Comment:
Because like this, the `margin-right` of the identicon can stay the same, even when the label `div` is replaced by an input for the editable version of this component. This `padding-left` is basically the padding that the label input has (including the border).